arrows/vxl/hashed_image_classifier_filter.cxx

```diff
@@ -57,10 +59,18 @@ ::load_model()
 {
   if( !model_loaded )
   {
-    if( !hashed_classifier.load_from_file( model_file ) )
+    auto const& model_paths = vital::find_config_file( model_file );
+    if( model_paths.empty() )
+    {
+      LOG_ERROR( p->logger(),
+                 "Could not locate \"" << model_file << "\" model" );
+      return false;
+    }
+
+    if( !hashed_classifier.load_from_file( model_paths.front() ) )
```

Comment:
I don't see a particular reason for this, but since we potentially have other paths, would it be useful to try to load from each of them? And then only error if none of them load properly.

Reply:
I'm not sure if it's desirable to skip a broken file and move on to one that works.  It's likely that error would go unnoticed and you'll end up running with a different file than expected without realizing.  It's probably better to fail and let the user figure out why they have a bad file on their path.